src/commands/registries/dockerHub/openDockerHubInBrowser.ts

```diff
@@ -22,7 +23,8 @@ export async function openDockerHubInBrowser(context: IActionContext, node?: Doc
     } else if (node instanceof DockerHubRepositoryTreeItem) {
         url += `r/${node.parent.namespace}/${node.repoName}`;
     } else {
-        url += `r/${node.parent.parent.namespace}/${node.parent.repoName}/tags`
+        const repoTI = <DockerHubRepositoryTreeItem>node.parent;
+        url += `r/${repoTI.parent.namespace}/${repoTI.repoName}/tags`
```

Comment:
Missing trailing `;`

Reply:
Yep common problem for this repo and I always forget. In other repos the tslint rule is on and it auto-fixes if you forget. I want to get this rule turned on and auto-fix the whole repo, but for now I'm not super worried about it